website/docs/addon_deadline_admin.md

```diff
@@ -90,6 +92,32 @@ For this admin needs to enable authentication requirement for DL in `ayon+settin
 
 Artists then need to provide their credentials in `Site Settings`.
 
+## Publish Plugins
+
+### Process submitted job on farm
+
+It submits a dependent AYON Job to publish the rendered images.
+- Find the setting at: `ayon+settings://deadline/publish/ProcessSubmittedJobOnFarm`
+
+![](assets/deadline/process_submitted_job_on_farm.png)
+
+1. **Department:**
+2. **Pool:**
+3. **Group:**
+4. **Chunk Size:**
+5. **Priority:**
+6. **Publishing script path:**
+7. **Skip integration of representation with ext:**
```

Comment:
Are the descriptions for these intentionally left empty @MustafaJafar ?

Reply:
I forgot to add them. Thanks for pointing that out.